src/cmdc/datasets/covidactnow/schema.sql

```diff
@@ -0,0 +1,209 @@
+CREATE TABLE meta.actnow_intervention_types (
+    id smallint primary key,
+    name text
+);
+
+INSERT INTO meta.actnow_intervention_types (id, name) VALUES
+(1, 'NO_INTERVENTION'), (2, 'WEAK_INTERVENTION'), (3, 'STRONG_INTERVENTION'), (4, 'OBSERVED_INTERVENTION');
+
+DROP TABLE IF EXISTS data.actnow_county_timeseries;
+CREATE TABLE data.actnow_county_timeseries (
+    "date" DATE,
+    "vintage" DATE,
+    "intervention_id" smallint references meta.actnow_intervention_types(id),
+    "fips" int references data.us_counties(fips),
+    "hospital_beds_required" INT,
+    "hospital_bed_capacity" INT,
+    "icu_beds_in_use" INT,
+    "icu_bed_capacity" INT,
+    "ventilators_in_use" INT,
+    "ventilator_capacity" INT,
+    "rt_indicator" numeric(12, 6),
```

Comment:
Why `NUMERIC(12, 6)` instead of just `FLOAT`?

Reply:
This is a habit of mine (and done automatically by the schema generator function).

NUMERIC will be handled exactly to the specified precision. (12, 6) means 12 digits, 6 after the decimal.

FLOAT would be handled like a floating point. It really shouldn't matter for this use case. I'm not sure what the storage requirements are for this compared to FLOAT

Reply:
My concern about `NUMERIC` is that exact precision computations are typically slower than `FLOAT` or `INTEGER` computations. If we just ingest to Pandas then this won't matter so much, but, if we plan to do computations on the back-end side, it may slow things down.

> The type numeric can store numbers with a very large number of digits. It is especially recommended for storing monetary amounts and other quantities where exactness is required. Calculations with numeric values yield exact results where possible, e.g. addition, subtraction, multiplication. However, calculations on numeric values are very slow compared to the integer types, or to the floating-point types described in the next section.

Source: https://www.postgresql.org/docs/10/datatype-numeric.html

Reply:
OK, I'm not picky either way. we can always convert to FLOAT inplace later on